tests/subsys/debug/coredump/prj.conf

```diff
@@ -1,4 +1,5 @@
 CONFIG_TEST=y
 CONFIG_DEBUG_COREDUMP=y
+CONFIG_LOG=y
```

Comment:
this fixes just this sample, a more generic fix maybe:

```diff
diff --git a/subsys/debug/coredump/Kconfig b/subsys/debug/coredump/Kconfig
index abeb0d3bda..7031fc6165 100644
--- a/subsys/debug/coredump/Kconfig
+++ b/subsys/debug/coredump/Kconfig
@@ -11,11 +11,11 @@ if DEBUG_COREDUMP

 choice DEBUG_COREDUMP_BACKEND
        prompt "Coredump backend"
-       default DEBUG_COREDUMP_BACKEND_LOGGING if LOG
+       default DEBUG_COREDUMP_BACKEND_LOGGING

 config DEBUG_COREDUMP_BACKEND_LOGGING
        bool "Use Logging subsystem for coredump"
-       depends on LOG
+       select LOG
        help
          Core dump is done via logging subsystem.

```

Reply:
good call -- updated.
Thoughts on adding guard(s) for `DEBUG_COREDUMP_BACKEND_INTEL_ADSP_MEM_WINDOW`?

Reply:
yeah, we can make this depend on DT_HAS_INTEL_ADSP_MEM_WINDOW_ENABLED

Reply:
```
 config DEBUG_COREDUMP_BACKEND_INTEL_ADSP_MEM_WINDOW
        bool "Use memory window for coredump on Intel ADSP"
+       depends on DT_HAS_INTEL_ADSP_MEM_WINDOW_ENABLED
        help
```

Reply:
thanks! done.